src/components/HeaderWithSearchBar/assets/shoppingCartIcon.vue

```diff
@@ -1,55 +1,48 @@
-          id="shopping-bag_1_"
-          data-name="shopping-bag (1)"
-          transform="translate(0)"
-        >
-          <g id="Grupo_11169" data-name="Grupo 11169" transform="translate(0)">
-            <g id="Grupo_11168" data-name="Grupo 11168">
-              <path
-                id="Trazado_31565"
-                data-name="Trazado 31565"
-                d="M76.793,24.739,75.453,7.858a2.11,2.11,0,0,0-2.094-1.94H70.712V5.8a5.8,5.8,0,1,0-11.6,0v.119H56.468a2.109,2.109,0,0,0-2.094,1.937l-1.34,16.886A4.885,4.885,0,0,0,57.9,30H71.922a4.885,4.885,0,0,0,4.87-5.261ZM60.872,5.8a4.042,4.042,0,1,1,8.083,0v.119H60.872ZM74.217,27.241a3.1,3.1,0,0,1-2.295,1H57.9a3.127,3.127,0,0,1-3.119-3.364l1.34-16.886a.345.345,0,0,1,.342-.317h2.646V9.839a.879.879,0,1,0,1.757,0V7.675h8.083V9.839a.879.879,0,0,0,1.757,0V7.675h2.646a.345.345,0,0,1,.342.319l1.34,16.882A3.1,3.1,0,0,1,74.217,27.241Z"
-                transform="translate(-53.02)"
-                stroke-width="1"
-              />
-            </g>
-          </g>
-          <g
-            id="Grupo_11171"
-            data-name="Grupo 11171"
-            transform="translate(7.678 14.642)"
-          >
-            <g
-              id="Grupo_11170"
-              data-name="Grupo 11170"
-              transform="translate(0)"
-            >
-              <path
-                id="Trazado_31566"
-                data-name="Trazado 31566"
-                d="M192.226,250.15a.878.878,0,0,0-1.242,0l-3.923,3.923-1.509-1.509a.879.879,0,0,0-1.243,1.243l2.13,2.13a.878.878,0,0,0,1.242,0l4.544-4.544A.879.879,0,0,0,192.226,250.15Z"
-                transform="translate(-184.052 -249.893)"
-                stroke-width="1"
-              />
-            </g>
-          </g>
-        </g>
+    <g>
+      <g>
+        <path
+          d="M276.199,342.977c-1.988,0-3.985-0.475-5.814-1.436l-94.908-49.896l-94.908,49.896c-4.212,2.213-9.315,1.845-13.164-0.952
+			c-3.85-2.796-5.777-7.536-4.973-12.227l18.125-105.682L3.775,147.836c-3.407-3.321-4.633-8.289-3.163-12.813
+			c1.47-4.525,5.382-7.823,10.091-8.508l106.111-15.418l47.454-96.153c2.105-4.267,6.451-6.968,11.209-6.968
+			c4.757,0,9.104,2.701,11.208,6.968l47.454,96.153l106.11,15.418c4.709,0.685,8.62,3.982,10.091,8.508
+			c1.471,4.524,0.245,9.492-3.163,12.813l-76.783,74.845l18.126,105.682c0.804,4.689-1.124,9.431-4.973,12.227
+			C281.37,342.172,278.792,342.977,276.199,342.977z M39.362,147.615l63.352,61.752c2.946,2.871,4.291,7.01,3.594,11.063
+			l-14.955,87.196l78.306-41.17c3.642-1.914,7.992-1.914,11.634,0l78.306,41.168l-14.956-87.194
+			c-0.695-4.056,0.649-8.192,3.595-11.063l63.353-61.752l-87.549-12.722c-4.071-0.592-7.591-3.148-9.412-6.838l-39.152-79.334
+			l-39.153,79.334c-1.82,3.689-5.341,6.246-9.412,6.838L39.362,147.615z"
+        />
       </g>
     </g>
+    <g></g>
+    <g></g>
+    <g></g>
+    <g></g>
+    <g></g>
+    <g></g>
+    <g></g>
+    <g></g>
+    <g></g>
+    <g></g>
+    <g></g>
+    <g></g>
+    <g></g>
+    <g></g>
+    <g></g>
```

Comment:
is this really needed?

Reply:
I think it's not needed

Reply:
Seems like it works without those <g> empties tags
![lastpeek](https://user-images.githubusercontent.com/66505715/208465591-583574c0-cdeb-479b-9cdf-35452fdb988c.gif)